spec/stepmod/converters/a_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "support/smrl_converters_setup"
+
+RSpec.describe Stepmod::Utils::Converters::A do
+  let(:converter) { described_class.new }
+
+  context "format with number" do
+    describe "<a href=\"#eqn7\">(71)</a>" do
+      let(:xml_input) do
+        '<a href="#eqn7">(71)</a>'
```

Comment:
Can you also help cover the case of `Figure <a ...>(1)</a>` Thanks!

Reply:
@HassanAkbar but we want to convert `Figure <a...>(1)</a>` into just `<<eqn1>>`?

Reply:
@ronaldtse  currently I’m converting `<a href="#fig1">Figure (1)</a>` to `<<fig1>>` by adding the `href` inside the `<<>>` tag. Isn’t this the correct behavior? Do I need to change this?

Reply:
I want to add one type of conversion in addition to what you have now:
```
Figure <a href="fig1">(1)</a>
```
=> 
```
<<fig1>>
```

Reply:
@ronaldtse So, from my understanding, we need to remove the prefix that appears before the `a` tag. Is this correct? If yes then which prefix do we need to remove? Only `Figure` and `Table` or are there some others?

Reply:
Yes. Figure Table Formula. I think we need to check if there are other prefixes.